src/plugins/intel_gpu/src/kernel_selector/cl_kernels/include/batch_headers/fetch_data.cl

```diff
@@ -683,9 +713,9 @@ inline uint get_bs_fs_zyx_bsv_fsv_index_safe(uint b, uint f, uint z, uint y, uin
     const uint z_pitch = y_pitch * (y_pad_before +  y_size + y_pad_after);
     const uint total_f_size = f_pad_before + f_size + f_pad_after;
     const uint fs_pitch = z_pitch * (z_pad_before +  z_size + z_pad_after);
-    const uint b_pitch = fs_pitch * ((total_f_size + alignmentF - 1) / alignmentF);
+    const uint bs_pitch = fs_pitch * ((total_f_size + alignmentF - 1) / alignmentF);
 
-    const uint output_offset = (bs * b_pitch) + (bsv * alignmentF) +
+    const uint output_offset = ((b_pad_before + bs) * bs_pitch) + (bsv * alignmentF) +
```

Comment:
I think `b_pad_before` should go into `b_mod`.
If `b_pad_before` go into `bs`, it would jump 32-batch for bsv32 format.

Reply:
You are right. It should be calculated to bs_pad_before/bsv_pad_before.

    const uint bsv_pad_before = b_pad_before % alignmentB;
    const uint output_offset = ((bs + bs_pad_before) * bs_pitch) +
                               (b_pad_before + fs) * fs_pitch +
                               (z_pad_before + (z % z_size)) * z_pitch +
                               (y_pad_before + (y % y_size)) * y_pitch +
                               (x_pad_before + (x % x_size)) * x_pitch +
                               ((bsv + bsv_pad_before) * alignmentF)
                               + fsv;